lib/diagnostic.js

```diff
@@ -19,16 +19,27 @@ normalizeWords(string);
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (string) {
+  return let arrayUniqueNormalizedWords = string.split(' ');
 };
 
+let string = "This is my string string";
+
+uniqueWords(string);
+
+// returns a array but not unique words yet
+
 // 3. Write a function that returns the count of words in a string.
 // Provid the *option* to count unique words instead of total words.
-const wordCount = function (/* */) {
-  /* your response here */
+const wordCount = function (string) {
+ let string = string.length;
 };
 
+wordCount("This is my sentence");
+console.log(string);
+
```

Comment:
You probably want to use your normalizeWords function here and then count the length of the array. What happens when we call .length on a string?